footprints/templates/clientside/search_template.html

```diff
@@ -1,7 +1,17 @@
 {% verbatim %}
 <script type="text/x-template" id="search-template">
     <div class="widget-pane">
-        <div class="widget-pane-content">
+        <div class="widget-pane-content" v-if="criteria">
+            <div v-if="total">
+                Book Copies: {{total}}
+            </div>
+            <form>
+                <select v-model="criteria.work">
+                    <option>-----</option>
+                    <option value="67">Akedet</option>
```

Comment:
@sdreher was this value meant to be inlined?

Reply:
a. Good catch. b. Yeah, this is okay to go to production. This is simply for some initial test purposes.